src/util/psbt/map/input.rs

```diff
@@ -191,6 +191,22 @@ impl PsbtSigHashType {
 }
 
 impl Input {
+    /// Returns sighash_type.ecdsa_hash_ty() is sighash_type exists else 
```

Comment:
The documentation can updated as

/// Obtains the [`EcdsaSigHashType`] for this input if one is specified. If no sighash type is specified, returns ...
///
/// Errors:
///     If the sighash type is not a standard ecdsa sighash type

Same comment for schnorr